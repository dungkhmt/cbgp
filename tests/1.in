20 20
20 49
8 19
6 5
3 7
19 5
5 2
5 10
8 5
15 19
8 2
2 3
11 13
9 1
12 12
12 14
18 10
5 11
16 4
6 18
13 5
10 17
5 3
2 2
11 10
12 2
2 7
2 4
16 3
19 10
8 15
20 1
1 2
9 20
12 20
2 20
2 16
10 6
