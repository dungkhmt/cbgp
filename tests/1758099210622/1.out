(16, 12), (8, 15), (10, 6), (14, 18), (5, 1), (2, 1), (0, 13), (2, 0), (6, 5), (3, 2), (3, 5), (14, 11), (14, 0), (11, 3), (4, 15), (3, 8), (7, 5), (11, 19), (8, 19), (6, 2), (11, 8), (14, 3), (9, 13), (4, 0), (3, 18), (0, 11), (3, 6), (9, 19), (0, 1), (0, 8), (7, 17), (17, 5), (18, 11), (10, 3), (0, 15), (0, 16), (0, 5), (12, 1), (4, 8), (14, 2), (2, 7), (0, 9), (3, 0), (4, 3), (0, 19), 
numberIntersection=0 
minAngle=0.013332543294145793 
sumAngle=14.569675639615784 
minEdgeLength=4.0 
sumEdgeLength=1260.0862789639314 
minDistance=0.40613846605344767 
sumDistance=19989.560722883598 

0.0000000000 0.0133325433 14.5696756396 4.0000000000 1260.0862789639 0.4061384661 19989.5607228836
0: (72, 0),
1: (32, 64),
2: (0, 0),
3: (40, 40),
4: (42, 32),
5: (32, 60),
6: (34, 44),
7: (26, 48),
8: (38, 24),
9: (36, 12),
10: (36, 48),
11: (30, 8),
12: (34, 68),
13: (38, 16),
14: (24, 4),
15: (42, 28),
16: (38, 68),
17: (32, 48),
18: (28, 12),
19: (38, 20),
72 0 32 64 0 0 40 40 42 32 32 60 34 44 26 48 38 24 36 12 36 48 30 8 34 68 38 16 24 4 42 28 38 68 32 48 28 12 38 20 
0: (72, 0), 1: (32, 64), 2: (0, 0), 3: (40, 40), 4: (42, 32), 5: (32, 60), 6: (34, 44), 7: (26, 48), 8: (38, 24), 9: (36, 12), 10: (36, 48), 11: (30, 8), 12: (34, 68), 13: (38, 16), 14: (24, 4), 15: (42, 28), 16: (38, 68), 17: (32, 48), 18: (28, 12), 19: (38, 20), 
numberIntersection=0 minAngle=0.013332543294145793 sumAngle=14.569675639615784 minEdgeLength=4.0 sumEdgeLength=1260.0862789639314 minDistance=0.40613846605344767 sumDistance=19989.560722883598 
Time taken: 41.689 s
